modules/polling/api/getPollTags.ts

```diff
@@ -0,0 +1,29 @@
+import { fsCacheGet, fsCacheSet } from 'lib/fscache';
+import { Tag } from 'modules/app/types/tag.dt';
+
+import pollTags from 'modules/tags/constants/poll-tags-definitions.json';
+import pollTagsMapping from 'modules/tags/constants/poll-tags-mapping.json';
+export function getPollTags(): Tag[] {
+  return pollTags;
+}
+
+export async function getPollTagsMapping(): Promise<{ [key: number]: string[] }> {
+  try {
+    const existingTags = fsCacheGet('poll-tags-mapping');
+
+    if (existingTags) {
+      return JSON.parse(existingTags);
+    }
+
+    const urlPollTags =
```

Comment:
@rafinskipg just verifying that this URL is correct. I know we have a bot to pull the changes from `makerdao/community/...` but there can be a short delay.